javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,97 @@
+// voice assistant 
+
+let userName= "";
+const todolist=[];
+
+function getReply(command){
+if (command==="Hello my name is Benjamin" && userName==null){
+    return `You are not a person`
+}
+    if (command ==="Hello my name is Benjamin" && userName===""){
+        userName="Benjamin";
+        return `Nice to meet you ${userName}`;
+
+    } if(command ==="Hello my name is Benjamin" && userName!=""){
+        userName="";
+        return`This name was already mentioned`;
+ 
+    } if (command==="what is my name?"){
```

Comment:
It will not work with `getReply("What is my name?");`

```suggestion
    } if (command === "What is my name?") {
```